backend/app/services/subscription/execution.py

```
@@ -69,11 +69,42 @@ def create_execution(
+        prompt_template = subscription_crd.spec.promptTemplate
+        display_name = subscription_crd.spec.displayName
+
+        if internal.get("is_rental", False):
+            source_subscription_id = internal.get("source_subscription_id")
+            if source_subscription_id:
+                source_subscription = (
+                    db.query(Kind)
+                    .filter(
+                        Kind.id == source_subscription_id,
+                        Kind.kind == "Subscription",
+                        Kind.is_active == True,
```

Ruff (0.14.13)

Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)